src/core/agent/services/multiSigService.ts

```diff
@@ -615,6 +617,46 @@ class MultiSigService extends AgentService {
     };
   }
 
+  async membersReadyToRotate(identifierId: string): Promise<string[]> {
+    const metadata = await this.identifierStorage.getIdentifierMetadata(
+      identifierId
+    );
+
+    const multiSig = await this.props.signifyClient
+      .identifiers()
+      .get(metadata.signifyName);
+
+    const members = await this.props.signifyClient
+      .identifiers()
+      .members(metadata?.signifyName);
+
+    const nextSequence = (Number(multiSig.state.s) + 1).toString();
+
+    const rmids = members?.rotation;
```

Comment:
I think we should use `smids` and `.signing`. And `states` instead of `rstates`.

I just took a deeper look at keripy and if we "abandon" an identifier by rotating to a null pre-rotated keys, `rmids` will be `[]`, which would break this.